lib/suse/connect/client.rb

```diff
@@ -34,11 +34,11 @@ def register!
         log.info 'Successfully registered system.'
       end
 
-      # Activate the product, add the service and install the relase package
+      # Activate the product, add the service and install the release package
       def register_product(product, install_release_package = true)
         service = activate_product(product, @config.email)
 
-        System.add_service(service)
+        System.add_service(service, !@config.no_zypper_refs)
```

Comment:
In general I dislike this double negation: Its harder to read than `zypper_refresh = false|true` especially in the spec files. But anyway in this case its fine.

Reply:
I know but default can be also `true`. Anyway I have no problem with this. Just more like a comment without an request to change :P